Makefile

```diff
@@ -377,6 +377,7 @@ test.integration.dbless:
 	@$(MAKE) _test.integration \
 		GOTAGS="integration_tests" \
 		DBMODE=off \
+		KONG_CONTROLLER_FEATURE_GATES="Knative=true" \
```

Comment:
This shouldn't be set: please note that this makefile target does not have the knative related build tags set so it will not run those anyway.

Reply:
fixed.